types/datatables.net-rowgroup/index.d.ts

```diff
@@ -63,7 +63,7 @@ declare namespace DataTables {
         /**
          * Set the data point to use as the grouping data source
          */
-        dataSrc?: number|string;
+        dataSrc?: number|string|string[];
```

Comment:
To confirm, it can be a `string[]`, but not a `number[]`?  (I know nothing about this library.)